plugin/mutate/source/dextool/plugin/mutate/backend/analyze/pass_schemata.d

```diff
@@ -84,8 +84,11 @@ uint checksumToId(Checksum cs) @safe pure nothrow @nogc {
 }
 
 uint checksumToId(ulong cs) @safe pure nothrow @nogc {
-    auto r = cast(uint) cs;
-    return r == 0 ? 1 : r;
```

Comment:
aaa add back this check for zero. it is very important because no ID is allowed to be zero. zero means "no mutant is active"